lib/ts/recipe/session/recipe.ts

```diff
@@ -118,6 +128,22 @@ export default class SessionRecipe extends RecipeModule {
         SessionRecipe.instance = undefined;
     }
 
+    static addClaimFromOtherRecipe = (builder: SessionClaim<any>) => {
+        SessionRecipe.claimsAddedByOtherRecipes.push(builder);
```

Comment:
I don't think we need to since this is only used internally, it should be easy to keep track of. If something was added twice it would only slow down session creation a bit (and it should be fairly minor at that).

Reply:
Well.. in that case, if we are passing in the same claim twice, we should throw an error - might help find bugs. I wouldn't want to just ignore this type of issue cause it's only meant for internal use.